mrtarget/common/ElasticsearchLoader.py

```diff
@@ -21,23 +20,13 @@ class Loader():
     """
 
     def __init__(self,
-                 es = None,
+                 es,
```

Comment:
@afaulconbridge same here, just passing the conf and let the class keeps the state internally is better in terms of scope and multiprocesses right?